main.nf

```diff
@@ -152,6 +155,10 @@ if( params.bwa_index ){
 } else {
     exit 1, "No reference genome specified!"
 }
+if( params.gtf ){
```

Comment:
Nice, speedy work! 👍 Maybe throw a warning if it isn't specified, to tell the user that steps `x` and `y` wil be skipped.

Reply:
I guess we can skip the warning for now. As only `GRCh37` and `GRCm38` are supported for peak calling, there will be a gtf file available as long as the genome is either of them.

Reply:
Ok, fair point. Note that we need to set `gtf = false` outside of this block to avoid an error due to the variable missing (the current travis failure).